README.md

```diff
@@ -13,13 +13,13 @@ The [Education Initiative](https://chanzuckerberg.com/education/)'s design syste
 
 ### Helpful commands
 
-| Description                           | Command                              |
-| ------------------------------------- | ------------------------------------ |
-| Install dependencies                  | `yarn install`                       |
-| Run linter                            | `yarn lint`                          |
-| Run linter and fix all fixable issues | `yarn lint:fix`                      |
-| Build packages                        | `yarn build`                         |
-| Run the component generator           | `yarn create-component`              |
```

Comment:
Wait didn't prettier just add this spacing in a different PR?

Reply:
I manually updated it on the Yarn PR, and probably Prettier didn't like it again. (not sure why I left in all the extra space)